database/models/User.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  DataType,
+  Column,
+  Model,
+  Table,
+  CreatedAt,
+  UpdatedAt,
+  BelongsToMany,
+} from "sequelize-typescript"
+
+import { Site } from "./Site"
+import { SiteMember } from "./SiteMember"
+
+@Table({ tableName: "users" })
+export class User extends Model {
+  @Column({
+    autoIncrement: true,
+    primaryKey: true,
+    allowNull: false,
+    type: DataType.BIGINT,
+  })
+  id!: number
+
+  @Column({
+    allowNull: true,
+    unique: true,
+    type: DataType.TEXT,
+    validate: {
+      isEmail: true,
+    },
+  })
+  email!: string
+
+  @Column({
+    allowNull: false,
+    unique: true,
+    type: DataType.TEXT,
+    validate: {
+      notEmpty: true,
+    },
+  })
+  githubId!: string
+
+  @Column({
+    allowNull: true,
+    type: DataType.STRING(255),
+  })
+  contactNumber!: string
+
+  @Column({
+    type: DataType.DATE,
+    defaultValue: DataType.NOW,
```

Comment:
i think this might be a little dangerous. we are explicitly setting the value when users login and if we allow creation of users by admins, might lead to an erroneous logged in time. 

should we just remove the default value?

Reply:
Let's follow up on [this thread](https://github.com/isomerpages/isomercms-backend/pull/376#discussion_r828747469) before returning to this discussion

Reply:
Defer decision till later. File an issue